src/azure-cli/azure/cli/command_modules/network/linter_exclusions.yml

```diff
@@ -25,6 +25,16 @@ network nsg rule update:
         direction:
             rule_exclusions:
             - missing_parameter_help
+network vnet create:
+    parameters:
+        private_endpoint_vnet_policies:
+            rule_exclusions:
+            - option_length_too_long
```

Comment:
Can we add a short option for this parameter instead of using linter exclusion?

Reply:
added option name as `--pe-vnet-policies`